plugins/modules/aci_domain_to_vlan_pool.py

```diff
@@ -330,21 +330,23 @@ def main():
             module_object=domain_mo,
             target_filter={"name": domain},
         ),
-        child_classes=["infraRsVlanNs"],
+        subclass_1=dict(
+            aci_class="infraRsVlanNs",
+            aci_rn="rsvlanNs",
+            module_object="rsvlanNs",
+            target_filter={"name": domain},
```

Comment:
Why did you change this back? I think we should mention the correct filters/module_object for consistency. Likely you would require below to have to all query ( class based query ) to succeed.

```suggestion
            module_object=aci_mo if pool_name else None,
            target_filter={"tDn": aci_mo} if pool_name else {},
```

Reply:
Hi. The query tests were failing, as it was trying to query the bindings without using any specific domain (query all domains) and therefore the target_filter with "tDn":aci_mo was empty
```
aci_mo = "uni/infra/vlanns-{0}".format(pool_name)
```

That was resulting in `uni/infra/vlanns-`
There is no way of quering "all bindings" on a single query without defining specifically which pool you want to query from, therefore i reverted back to query all phydoms, which contains all bindings